### PR TITLE
Add test to picker dialog

### PR DESCRIPTION
**Pull request by FBerthelot**
## JIRA

https://jira.jahia.org/browse/PROJECT-XXX

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/MainPanel.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    });
+
+    it('should display the title set in the picker configuration', () => {
+        const cmp = shallowWithTheme(
+            <MainPanel {...props}/>,
+            {},
+            dsGenericTheme
+        ).dive();
+
+        expect(cmp.debug()).toContain('');
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/MainPanel.spec.jsx]: Question: I don't understand the case, where is the title that should be displayed?

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            siteNodes: [],
+            lang: 'fr',
+            nodeTreeConfigs: [{}],
+            onSelectSite: jest.fn(),
+            selectedPath: '/selectedPath',
+            setSelectedPath: jest.fn()
+        };
+    });
+
+    it('should display siteSwitcher when there is multiple site to show', () => {
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx]: Minor: the message of the case it seems not a good fit, maybe we should updated it to understand the case that we talk about the special case when we hide/show the site switcher component.

Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx]: I confess that I don't understand well the behaviour of this.  Can you propose a better message as you are the one that implement it ? :)

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/MainPanel.spec.jsx (hunk 1 above)
Comment (FBerthelot) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/MainPanel.spec.jsx]: You are right this test is wrong 😨

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx (hunk 2 above)
Comment (Taybou) [reply src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx]: should display siteSwitcher when the picker is not of type "site picker"

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        const cmp = shallowWithTheme(
+            <LeftPanel {...props}/>,
+            {},
+            dsGenericTheme
+        ).dive();
+
+        expect(cmp.find('SiteSwitcher').exists()).toBe(true);
+    });
+
+    it('should not display siteSwitcher when there is no site to select', () => {
```
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx]:
```suggestion
    it('should hide siteSwitcher when the picker is of type "site picker"', () => {
```

File: src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx (hunk 2 above)
Comment (Taybou) [inline src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx]:
```suggestion
    it('should display siteSwitcher when the picker is not of type "site picker"', () => {
```

**Pull request merged**